Apps/phcybereason/cybereason_connector.py

```diff
@@ -0,0 +1,686 @@
+
+        except Exception as e:
+            return action_result.set_status(phantom.APP_ERROR, e)
+
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    def _handle_kill_process(self, param):
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # Access action parameters passed in the 'param' dictionary
+        malop_id = self._get_string_param(param["malop_id"])
+        machine_id = self._get_string_param(param["machine_id"])
+        remediation_user = param["remediation_user"]
+        process_id = self._get_string_param(param["process_id"])
+
+        try:
+            cr_session = CybereasonSession(self).get_session()
+            endpoint_url = "/rest/remediate"
+            url = self._base_url + endpoint_url
+            api_headers = {'Content-Type': 'application/json'}
+            query = {
+                "malopId": malop_id,
+                "initiatorUserName": remediation_user,
+                "actionsByMachine": {
+                    machine_id: [
+                        {
+                            "targetId": process_id,
+                            "actionType": "KILL_PROCESS"
+                        }
+                    ]
+                }
+            }
+            res = cr_session.post(url, json=query, headers=api_headers)
+            result = res.json()
+            if len(result["statusLog"]) > 0:
+                action_result.add_data({
+                    "remediation_id": result["remediationId"],
+                    "remediation_status": result["statusLog"][0]["status"]
+                })
+        except Exception as e:
+            self.debug_print(str(e))
+            self.debug_print(traceback.format_exc())
+            return action_result.set_status(phantom.APP_ERROR, e)
+
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    def _handle_get_remediation_status(self, param):
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # Access action parameters passed in the 'param' dictionary
+        malop_id = self._get_string_param(param["malop_id"])
+        remediation_user = param["remediation_user"]
+        remediation_id = param["remediation_id"]
+
+        try:
+            cr_session = CybereasonSession(self).get_session()
+            endpoint_url = "/rest/remediate/progress/" + remediation_user + "/" + malop_id + "/" + remediation_id
+            url = self._base_url + endpoint_url
+            res = cr_session.get(url)
+            self.debug_print(res.text)
```

Comment:
This could be a security vulnerability. Please make sure debug statements do not leak information